src/Error/Debug/HtmlFormatter.php

```diff
@@ -53,11 +53,7 @@ public function __construct()
      */
     public static function environmentMatches(): bool
     {
-        if (PHP_SAPI === 'cli' || PHP_SAPI === 'phpdbg') {
-            return false;
-        }
-
-        return true;
+        return !(PHP_SAPI === 'cli' || PHP_SAPI === 'phpdbg');
```

Comment:
Surprised this one isn't turned into != && != like the other PR.

Reply:
maybe it will, after the rules will run at a same time